src/udev/udev-rules.c

```diff
@@ -1849,9 +1849,11 @@ static int udev_rule_apply_token_to_event(
                 }
 
                 r = strv_split_newlines_full(&lines, result, EXTRACT_RETAIN_ESCAPE);
-                if (r < 0)
+                if (r < 0) {
```

Comment:
this could also return EINVAL, no? Shouldn't it check explicitly for ENOMEM before returning?